releases/release-1.23/release-notes/maps/pr-104877-map.yaml

```diff
@@ -0,0 +1,4 @@
+pr: 104877
+releasenote:
+  text: Kubernetes object references (= name + namespace) were not logged as struct
+    when using JSON as log output format.
```

Comment:
Should be: Migrated kubernetes object references (= name + namespace) to structured logging.

Reply:
"Migrated kubernetes object references (= name + namespace) to structured logging when using Json as log output format" or just that?

Reply:
Let's go with "Migrated kubernetes object references (= name + namespace) to structured logging when using JSON as log output format". Thank you^